cmd/test-image/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM nicolaka/netshoot
+
```

Comment:
A lot of the test operations in 'main' exist in tools in nicolaka/netshoot.  For example: iperf, netgen, netcat, drill, ping, etc.  Given that there is an /exec/ method that can invoke them, should some of these seem kind of redundant.  (icmp, tcp_ping, etc..)  I guess there's no real harm in them.  Just an observation.

Reply:
the idea is that this image is for test and also debug. The reason why is based on netshoot is that in case is needed a live debug all the networking tools are already there. Also the `/exec` is at the moment just returning the output of the command, so the server in case will require to parse it to make sense out of it. Let's say that in my mind `/exec` is there for convenience on debug cases.